src/ReleaseAction/GitHubReleaseAction.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+namespace Leviy\ReleaseTool\ReleaseAction;
+
+use Leviy\ReleaseTool\GitHub\GitHubClient;
+use Leviy\ReleaseTool\Vcs\VersionControlSystem;
+
+final class GitHubReleaseAction implements ReleaseAction
+{
+    /**
+     * @var GitHubClient
+     */
+    private $client;
+
+    /**
+     * @var VersionControlSystem
+     */
+    private $vcs;
+
+    public function __construct(GitHubClient $client, VersionControlSystem $vcs)
```

Comment:
Should it be possible to use the `GitHubReleaseAction` with a version control system other than Git? Say, if we were to add SVN as a VCS option, would it make sense to use it with the GitHub release action?

Reply:
I'm not familiar with SVN/Mercurial etc. after a short google it seems you can use SVN with GitHub but i don't think it supports releases... Not sure about this though.